Add `querySelectorAll()` examples for `Using data attributes` page (#38049)
mdn/content: files/en-us/learn_web_development/howto/solve_html_problems/use_data_attributes/index.md

Each property is a string and can be read and written. In the above case setting `article.dataset.columns = 5` would change that attribute to `"5"`.

> You can also use [`querySelectorAll()`](/en-US/docs/Web/API/Document/querySelectorAll) with data attribute selectors to find all elements that have a specific data attribute or value:

Reviewer: ```suggestion
You can also use [`document.querySelector()`](/en-US/docs/Web/API/Document/querySelector) or [`document.querySelectorAll()`](/en-US/docs/Web/API/Document/querySelectorAll) with data attribute selectors to find one element or all elements that match:
```

```js
// Find all elements with a data-columns attribute
const articles = document.querySelectorAll("[data-columns]");

// Find all elements with data-columns="3"
const threeColumnArticles = document.querySelectorAll('[data-columns="3"]');

> 
> // Find elements with a data-parent attribute containing "cars"
> const carRelated = document.querySelectorAll('[data-parent="cars"]');
> 

Reviewer: ```suggestion

```
 
Since it's basically the same as the one above
